src/App.tsx

```diff
@@ -12,7 +12,7 @@ const Container = styled.div`
 const HARDCODED_USER = {
   firstName: "John",
   lastName: "Doe",
-  email: "john@doe.com",
+  email: "john@doedoedoedoedoedoe.com",
```

Comment:
```suggestion
  email: "john@doe.com",
```
Just restore the dummy user data

Reply:
Removed these files from this commit, this was an accident. Cheers Simon